app/build.gradle

```diff
@@ -7,8 +7,8 @@ apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 apply plugin: 'realm-android'
 
-def STATIC_API_KEY = '"'+StaticTokenKey+'"' ?: '"Define your static api key"'
 def YOUTUBE_API_KEY = System.getenv('YOUTUBE_API_KEY') ?: "YOUR_API_KEY"
+def STATIC_API_KEY = '"'+System.getenv('MY_TOKEN')+'"' ?: '"DEFAULT"'
```

Comment:
What's my token?

Reply:
Token's Variable name in system environment varaibles

Reply:
What should I put in `MY_TOKEN` environment variable?

Reply:
Token for Static map generated by any admin of Fossasia organization.

Reply:
So it should be named GOOGLE_MAPS_API_KEY ?

Reply:
It is not generated from Google. I use service of mapbox for static map.

Reply:
But if you say I can set format for Google static Map API

Reply:
Change this to MAPBOX_API_KEY then